sklearn/ensemble/tests/test_gradient_boosting.py

```diff
@@ -35,6 +35,12 @@
 from sklearn.svm import NuSVR
 
 
+pytestmark = pytest.mark.filterwarnings(
+    # TODO: Remove when issue #22458 is resolved
+    "ignore:`max_features='auto'` has been deprecated in 1.1"
```

Comment:
I do not think we need the per file filterwarning here. In this case, `test_max_feature_auto` is the only test that needs the decorator.

Reply:
Both `test_tree.py` and `test_gradient_boosting.py` updated as suggested.